### PR TITLE
Light polish for the animation chapter

### PR DESCRIPTION
**Pull request by BrianSipple**
I made a few polishes for the [animation chapter](http://openframeworks.cc/ofBook/chapters/animation.html) as I went through it. 

Outside of my editor deleting a few extra spaces in some places, one change is a terminology fix and the other is a punctuation fix.

Cheers!

### CODE REVIEW

File: chapters/animation/chapter.md [hunk 1]
```diff
@@ -374,11 +374,11 @@ Finally, if we alter the angles we pass in to x and y for this formula at differ
...
 
 When I say continuous function, what I mean is if you pass in smaller changes as input, you get smaller output and if you pass in the same value you get the same result. For example, `sin(1.7)` always returns the same value, and `ofNoise(1.7)` also always returns the same result. Likewise if you call `sin(1.7)` and `sin(1.75)` you get results that are continuous (meaning, you can call `sin(1.71) sin(1.72)... sin(1.74)` to get intermediate results).
 
-You can do the same thing with ofNoise -- here, I write a for loop to draw noise as a line. `ofNoise` takes an input, here i/10 and produces an output which is between 0 and 1.  [`ofSignedNoise`](http://openframeworks.cc/documentation/math/ofMath.html#!show_ofSignedNoise "ofSignedNoise Documentation Page") is similar but it produces an output between -1 and 1.
+You can do the same thing with ofNoise -- here, I write a for loop to draw noise as a line. `ofNoise` takes an input -- here, i/10 -- and produces an output which is between 0 and 1.  [`ofSignedNoise`](http://openframeworks.cc/documentation/math/ofMath.html#!show_ofSignedNoise "ofSignedNoise Documentation Page") is similar but it produces an output between -1 and 1.
```
Comment (mikewesthad) [inline chapters/animation/chapter.md]: This reads a little awkwardly to me. What about:

> You can do the same thing with ofNoise. Here, I write a for loop to draw noise as a line. `ofNoise` takes an input (i/10 in this case) and produces an output which is between 0 and 1.

**Comment by mikewesthad** [comment]
This looks great to me - thanks again @BrianSipple for the edits. There was one edit where I had an additional suggestion. (Pinging the author, @ofZach, to let him know about the PR.)

File: chapters/animation/chapter.md (hunk 1 above)
Comment (BrianSipple) [reply chapters/animation/chapter.md]: You dashed my em dash dreams, @mikewesthad. But I like that that, too. Just updated and rebased.

Comment (mikewesthad) [reply chapters/animation/chapter.md]: Clever 👍

**Comment by mikewesthad** [comment]
This looks good to me - so I'm going to go ahead and merge it in. @ofZach can reopen/revert if needed.

**Comment by ofZach** [comment]
hey sorry but I'd really like to review changes to my chapter

**Comment by mikewesthad** [comment]
@ofZach - no problem, reverted. Sorry! Just trying to make sure we don't accumulate stale issues & PRs when the scope of the changes is small.

**Comment by ofZach** [comment]
thanks -- I was traveling and didn't have a chance to look at this...   

I wouldn't change for loop to for-loop (which seems wrong to me) but besides that it seems ok to me and they are small changes.  thanks for giving me a chance to check this out.

**Comment by ofZach** [comment]
also, thanks! @BrianSipple for fixes!

**Comment by BrianSipple** [comment]
@mikewesthad It looks like reverting the changes doesn't also reopen the PR. I'll open a new one for @ofZach to get some 👀  on (while also addressing the [feedback](https://github.com/openframeworks/ofBook/pull/270#issuecomment-337360669)).

**Comment by mikewesthad** [comment]
@BrianSipple - sorry about that! Thanks for opening a new one

**Pull request merged**